examples/tone_conversation_integration.v1.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * Tone Analyzer integration with Conversation
+ * 
+ * This example demonstrates how to detect and interpret the tone for a user's input (a conversation turn) 
+ * using the Tone Analyzer Service, and add it to the payload to be sent in a request to the Conversation 
+ * Service API message endpoint.  
+ * 
+ * Service credentials can be provided directly in this file, or can be saved to a .env file located in the 
+ * same directory.
+ * 
+ * Requirements:
+ *   1. Tone Analyzer Service instance: 
+ *   	- https://console.ng.bluemix.net/catalog/services/tone-analyzer/
+ *      - credentials for this service to be provided below in tone_analyzer variable
+ *      	- replace <tone_analyzer_username> and <tone_analyzer_password> 
+ *   2. Conversation Service instance: 
+ *   	- https://console.ng.bluemix.net/catalog/services/conversation/
+ *   	- credentials for this service to be provided below in the conversation variable
+ *   		- replace <conversation_username> and <conversation_password>
+ *   3. Workspace id: 
+ *   	- a workspace containing intents, entities and dialog nodes must be created using the tool 
+ *        available through the Bluemix Conversation Service.  Details are available at 
+ *        https://github.com/watson-developer-cloud/conversation-simple#workspace
+ *      - replace <workspace_id> in the payload variable
+ * 
+ * Run the code using the command:
+ *   node tone_conversation_integration.v1.js
+ */
+
+var watson = require('watson-developer-cloud');
+var tone_detection = require("./conversation_addons/tone_detection.js");
+require('dotenv').config({silent: true})
+
+/**
+ * Instantiate the Watson Conversation Service
+ */
+var conversation = watson.conversation({
+  url: 'https://gateway.watsonplatform.net/conversation/api',
+  username: process.env.CONVERSATION_USERNAME || '<conversation_username>',
+  password: process.env.CONVERSATION_PASSWORD || '<conversation_password>',
+  version_date: '2016-07-11',
+  version: 'v1'
+});
```

Comment:
FYI, you don't need to change anything here, but I made a updates to the SDK with v2 such that you can now create instances via `new watson.ConversationV1({/*...*/})`, and/or require the ConversationV1 file directly. 

I also changed it to automatically check those exact env properties if no credentials were passed in.

But, it's fully backwards compatible with everything here, so there's no problem with this example. (And explicitly calling out the env properties is probably good in an example.) I just thought I'd call it out since I'm going through and updating the docs and other examples this week.